apps/heureka/src/components/Services/ServicePanel/ServicePanel.tsx

```diff
@@ -109,7 +103,10 @@ export const ServicePanel = ({ service, isLoading = false, onClose }: ServicePan
               <DataGridRow key={index}>
                 <DataGridCell className="break-all overflow-hidden">
                   <Stack gap="1" direction="vertical">
-                    <span>{version.imageName}</span>
+                    <Stack gap="0.5" direction="vertical">
+                      <span>{version.imageName}</span>
+                      <span className="text-sm text-jn-global-text-subdued">{version.imageVersion}</span>
```

Comment:
The text color `text-jn-global-text-subdued` doesn't exist. What you want is `text-theme-light`.

Also **very important**: Never use classnames prefixed with `jn-` those are internal to juno-ui-components and it is not guaranteed that they work. instead only use the classname without the `jn-` prefix!!!